server/controller/routes/postAnswers.js

```diff
@@ -0,0 +1,23 @@
+const addAnswer = require("../../db/queries/postAnswers");
+
+module.exports = (req, res, next) => {
+  try {
+
+    // this is if we send all the answers as a object in the post body
+    // the structure needs to be { answer1, answer2, answer3 } = req.body.data;
+
+    const dataToSend = req.body.data;
+
+    addAnswer(dataToSend).then(result => {
```

Comment:
You send the data to the database without validation?

Reply:
I think that should be an issue in itself, one we tackle next week. I think we'll have time to add the extra questions from the feedback Josh gave us. So the structure of our input into the database will change